vars/main.yml

```diff
@@ -4,3 +4,4 @@
 rke2_method: tar
 rke2_service_name: "rke2-{{ rke2_type }}.service"
 rke2_restart_needed: false
+keepalived_secret: "{{ lookup('community.general.random_string', length=80, base64=True) }}"
```

Comment:
This will cause that all keepalived nodes will have different password

Reply:
Do you know if there is a way to cache the value or something so it's the same for all hosts? I guess it could be set as a fact first and then that fact could be used later?

Reply:
Technically you could use local facts for storing it. But then you would need solve also things like to have this secret idempotent but still have an option to change it, use it also for new node joining the keepalived cluster... etc. The solution would have to be too much programmatic i guess... And Ansible is not an programming language :-)  I think the value should be left up to user.